### DISCUSSION TITLE
No connection with network name FRITZ!Box 2,4

### DISCUSSION THREAD

**Discussion by johi-1** [tbnobody/OpenDTU#454] [General]
Open DTU works fine with it#s own network (OpenDTU****) I can go in the setup and put in my credentials in. I can save them. But it doesn't connect to the network I put in. Network info shows disabled. No name set, but in settings I can see network name and that a password was set.

**Comment by tbnobody** [comment]
Please show some screenshots

**Comment by johi-1** [comment]
Additional info: AHOY DTU runs fine with These Network settings




















Gesendet von Mail für Windows

Von: tbnobody
Gesendet: Freitag, 16. Dezember 2022 21:29
An: tbnobody/OpenDTU
Cc: johi-1; Author
Betreff: Re: [tbnobody/OpenDTU] No connection with network name FRITZ!Box 2,4(Discussion #454)

Please show some screenshots

**Comment by johi-1** [comment]
Everything is fine now, sorry. Looks like I wasn’t able to type in the SSID correctly. ☹

Gesendet von Mail für Windows

Von: tbnobody
Gesendet: Freitag, 16. Dezember 2022 21:29
An: tbnobody/OpenDTU
Cc: johi-1; Author
Betreff: Re: [tbnobody/OpenDTU] No connection with network name FRITZ!Box 2,4(Discussion #454)

Please show some screenshots
